apps/api/env/.env.functional.test

```
@@ -1,8 +1,8 @@
 POSTGRES_URI=postgres://postgres:password@localhost:5432
 UAKT_TOP_UP_MASTER_WALLET_MNEMONIC="since bread kind field rookie stairs elephant tent horror rice gain tongue collect goose rural garment cover client biology toe ability boat afford mind"
 USDC_TOP_UP_MASTER_WALLET_MNEMONIC="leaf brush weapon puppy depart hockey walnut hospital orphan require unfair hunt ribbon toe cereal eagle hour door awesome dress mouse when phone return"
-NETWORK=sandbox
-RPC_NODE_ENDPOINT=https://rpc.sandbox-2.aksh.pw:443
+NETWORK=testnet
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NETWORK key should go before the POSTGRES_URI key

(UnorderedKey)